python/README.md

```diff
@@ -0,0 +1,15 @@
+# Python Examples
+Python Pub/Sub Examples for HPE Ezmeral Data Fabric Topics
+
+## Install kafka-python: pip3 install kafka-python
+`pip3 install kafka-python`
+`pip3 list | grep kafka-python`
+
+### Running Examples
+### [Producing Messages](src/main/Produce.py)
```

Comment:
What is going to happen with the producer? Presumably it will send message, but what messages? Where will they be sent? How many?